web/src/configure-activity.css

```diff
@@ -44,6 +44,8 @@
     margin-right: 10px;
     border: 1px solid transparent;
     color: var(--interactive-medium-default);
+    background: none;
+    font-family: monospace;
```

Comment:
Changed this element from a `div` to a `button` to address lint errors related to accessibility. That affected its appearance so I added these styles which I think brings it back in line with the original appearance.